docs/API_ref.md

```diff
@@ -1,70 +1,78 @@
+| supplementalGroups | [][int64](https://pkg.go.dev/builtin#int64)    | SupplementalGroups defines the linux groups to be applied to the Restic Pod |
+| timeout            | [string](https://pkg.go.dev/builtin#string)     | Timeout defines the Restic timeout, default value is 1h                     |
+| PodConfig          | *[PodConfig](https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#PodConfig) | Restic Pod specific configuration                                           |
+
+### [PodConfig](https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#PodConfig)
 
 | Property            | Type                                                                                      | Description                                                                                                                               |
 |---------------------|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
-| nodeSelector        | map [ string ] string                                                                     | NodeSelector defines the nodeSelector to be supplied to Velero/Restic podSpec                                                             |
-| tolerations         | [[]corev1. Toleration](https://pkg.go.dev/k8s.io/api/core/v1#Toleration)                  | Tolerations defines the list of tolerations to be applied to Velero Deployment/Restic daemonset                                                             |
+| labels        | map [ [string](https://pkg.go.dev/builtin#string) ] [string](https://pkg.go.dev/builtin#string)                                                                     | Labels to add to pods                                                             |
+| nodeSelector        | map [ [string](https://pkg.go.dev/builtin#string) ] [string](https://pkg.go.dev/builtin#string)                                                                     | NodeSelector defines the nodeSelector to be supplied to Velero/Restic podSpec                                                             |
+| tolerations         | [][corev1.Toleration](https://pkg.go.dev/k8s.io/api/core/v1#Toleration)                  | Tolerations defines the list of tolerations to be applied to Velero Deployment/Restic daemonset                                                             |
 | resourceAllocations | [corev1.ResourceRequirements](https://pkg.go.dev/k8s.io/api/core/v1#ResourceRequirements) | Set specific resource  `limits`  and  `requests`  for the Velero/Restic pods. For more information, go  [here] ( config/resource_req_limits.md). |
 
-### Features
+### [Features](https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#Features)
```

Comment:
Can you please update the dataMover API as well ? Ref: https://github.com/openshift/oadp-operator/blob/master/bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml#L446